alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/destination/email/EmailAccount.kt

```diff
@@ -0,0 +1,134 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.alerting.model.destination.email
+
+import com.amazon.opendistroforelasticsearch.alerting.util.IndexUtils.Companion.NO_SCHEMA_VERSION
+import org.elasticsearch.common.settings.SecureString
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParser.Token
+import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import java.io.IOException
+
+/**
+ * A value object that represents an Email Account. Email Accounts contain the configuration
+ * information for sender emails when sending email messages through the Email destination.
+ */
+data class EmailAccount(
+    val id: String = NO_ID,
+    val schemaVersion: Int = NO_SCHEMA_VERSION,
+    val name: String,
+    val email: String,
+    val host: String,
+    val port: Int,
+    val method: MethodType,
+    val username: SecureString? = null,
+    val password: SecureString? = null
+) : ToXContent {
```

Comment:
Make it Writeable?

Reply:
I'll make `EmailAccount` and `EmailGroup` writeable when the REST API are updated. I think I at least changed `Email` since `Destination` was using it.